### ISSUE TITLE
Can't import for some reason

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [pyricau/frenchtoast#3]
I get this error:

Error:(20, 0) CreateProcess error=2, The system cannot find the file specified
<a href="openFile">Open File</a>

**Comment by pyricau** [comment]
Please provide more details. Closing.

**Comment by AndroidDeveloperLB** [comment]
more details about what?
That's the only error that I get...
I just import via Android-Studio, by using "VCS"->"Checkout..."->"GIT", and then I type the repo URL:
    https://github.com/pyricau/frenchtoast.git

and then it fails:

![untitled20150912170344](https://cloud.githubusercontent.com/assets/5357526/9831860/437ac6ca-5970-11e5-87af-81aa61776fb9.png)

here's android studio info:

Android Studio 1.4 Beta 3
Build #AI-141.2240497, built on September 9, 2015
JRE: 1.8.0_51-b16 amd64
JVM: Java HotSpot(TM) 64-Bit Server VM by Oracle Corporation

Why close this issue?

**Comment by pyricau** [comment]
Because there wasn't enough info or clarity in the bug description.

Not sure what to do here anyway. Happy to look at a PR if you come up with a fix.

**Comment by AndroidDeveloperLB** [comment]
Why is it still closed? What more info do you need? I can make a video if you wish...
Please explain what's wrong. What did I do wrong here?

**Comment by pyricau** [comment]
First comment lacked info. Thanks for adding more details. However, like I wrote, I have no idea how to fix this. Since it's not a problem for users of the library (who would normally add a Gradle dependency), only for people who want to checkout the repo, then it's not a priority for me. It's still closed because, if there's nothing I can do, and it's not important for users, then it's pointless to keep it open.

Now, like I wrote, if you come up with a PR to the setup files that fixes, I'm happy to review.
